runtime/proto/corfu_options.proto

```diff
@@ -11,10 +11,22 @@ message SchemaOptions {
     optional bool secondary_key = 1;
     // Version number in metadata field.
     optional bool version = 2;
+    // Should this table be backed up by Corfu.
+    optional bool needs_backup = 3;
```

Comment:
could we close on the convention needs_backup or requires_backup.

Reply:
we can go with requires_backup